backend/api/models.py

```
@@ -1,3 +1,120 @@
+    ADMIN = 'Admin'
+    COMPANY = 'Company'
+    USER = 'User'
+    ROLE_CHOICES = [
+        (ADMIN, 'Admin'),
+        (COMPANY, 'Company'),
+        (USER, 'User'),
+    ]
+
+    name = models.CharField(max_length=50, unique=True, choices=ROLE_CHOICES)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)